examples/asyncapi-from-parser/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "asyncapi-from-parser",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/asyncapi/modelina",
+  "bugs": {
+    "url": "https://github.com/asyncapi/modelina/issues"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.15",
+    "cross-env": "^7.0.2",
+    "jest": "^26.6.3",
+    "ts-jest": "^26.4.4",
+    "ts-node": "^10.2.1"
+  },
+  "scripts": {
+    "start": "ts-node index.ts",
+    "test": "cross-env CI=true jest --config=../../jest.config.js ./examples/asyncapi-from-object/index.spec.ts"
```

Comment:
do we need this script? and all the `dependencies` and `devDependencies` if at the we have `"test:examples": "cross-env CI=true jest ./examples",` in `package.json` + these dependencies are already there in the root?

can't we have just:
```json
{
  "scripts": {
    "start": "ts-node index.ts"
   }
}

Reply:
Because I want users to stand inside the example repo, and write `npm run start` to run the example as well as tests `npm run test`. That won't work if it has no dependencies defined. 

And I dont want to overrun the root package.json with scripts.

Reply:
I just cloned your branch, did `npm install` in root only, then modified the package.json:

```json
{
  "scripts": {
    "start": "ts-node index.ts",
    "test": "cross-env CI=true jest --config=../../jest.config.js ./examples/asyncapi-from-object/index.spec.ts"
  }
}
```

and then `npm run test` works like a charm.

for instructions it is fine if user needs to install root project imho, otherwise just add another script `"install": "cd ../.. && npm i"` if you want to hide complexity. Just having these dependencies all over the place will complicate things a lot, management of dependencies.

Reply:
Ahh right, cause the test script actually uses root dependencies. 

Did you try for the `asyncapi-from-parser` example, as it uses the parser dependency?

Reply:
works like a charm, parser is at the root of the project, so in `node_modules`. why you think it should not work?

Reply:
I dont get the setup you have, cause the `npm run start` cannot work with root dependencies and I dont like the requirement that you have to install `ts-node` globally, unless you have a setup I can't think of 😄

Can you provide the full example so you inside the example folder can successfully run `start` and `test` without defining dependencies for the example? 🤔 That also does not require `ts-node` to be globally installed separately.

Reply:
`ts-node` must be added in the project root to `devDependencies`. It is a `devDependency` and I think it is completely fine approach to have it there

Reply:
Now it snapped together how to achieve it, updated it 😄